### DISCUSSION TITLE
One list

### DISCUSSION THREAD

**Discussion by brend8c** [darkreader/darkreader#6757] [Ideas]
Might you create one list? 
One list for those sites that need to apply a dark theme.
I think the second list is not needed.
![2021-09-14 01_35_24-](https://user-images.githubusercontent.com/61526280/133165253-7b9ccccd-c632-4d71-b707-741585a29bb7.png)
I go to the site > If the site is on the list > The site gets a dark theme. Isn't it easier?

**Comment by Gusted** [comment]
That isn't how the sitelist should sussposed to be, but it's already a issue #28

**Reply by brend8c** [reply]
Fine!
